src/Pipeline/Span/Primary.php

```diff
@@ -27,29 +13,35 @@
 use Driver\Pipeline\Transport\Status;
 use Driver\Pipeline\Transport\TransportInterface;
 use Driver\System\YamlFormatter;
-use Haystack\HArray;
-use Symfony\Component\Console\Output\OutputInterface;
+
+use function array_filter;
```

Comment:
@michalbiarda Should we import PHP functions here?

Reply:
Actually, importing PHP functions (or using `\function_name` inline) is a good practice. That way you are always sure that you're using function from correct namespace (global in that case).

Reply:
Thanks for clarifying.